### PR TITLE
add order history view

### PR DESCRIPTION
**Pull request by NessEFC**

### CODE REVIEW

File: db/migrate/20170412235841_create_order_escapes.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class CreateOrderEscapes < ActiveRecord::Migration[5.0]
+  def change
+    create_table :order_escapes do |t|
+      t.references :order, index: true, foreign_key: true
+      t.references :escape, index: true, foreign_key: true
+      t.integer :quantity
```
Comment (caward12) [inline db/migrate/20170412235841_create_order_escapes.rb]: quantity and total will come from cart

File: spec/features/user_can_view_past_orders_spec.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.feature "As a logged-in user" do
+  scenario "they can view past orders" do
+
+    user = create(:user)
+    order_1 = user.orders.create
```
Comment (NessEFC) [inline spec/features/user_can_view_past_orders_spec.rb]: We need to figure out how to use factory girl to create these in the future.

File: app/models/order.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class Order < ApplicationRecord
```
Comment (lukyans) [inline app/models/order.rb]: We need to add validation

File: app/models/order_escape.rb [hunk 4]
```diff
@@ -0,0 +1,4 @@
+class OrderEscape < ApplicationRecord
```
Comment (lukyans) [inline app/models/order_escape.rb]: We need to add validation

**Pull request merged**